### ISSUE TITLE
how to use with ffmpeg mpegts websocket

### ISSUE THREAD

**Issue by andreac** [samirkumardas/jmuxer#56]
Hi, i'm working on a project where i use ffmpeg to stream mpegts segments to my local server thant send to frontend via websocket (using mpegjs)

I want to switch to jmuxer to check if performance are better. can you help me in convert video and audio and pass to websocket?

at the moment i'm using this command for ffmpeg

`-y -threads 8 -thread_queue_size 8096 -re -f rawvideo -vcodec rawvideo -pixel_format bgra -video_size 1024x576 -i pipe:0 -i ${payload.source} -filter_complex [1:v][0:v]overlay[out] -map [out] -vcodec libx264 -map 1:a:0 -f mpegts -codec:v mpeg1video -b:v 700k -q 1 http://localhost:3000/mystream`

basically i send raw frames to ffmpeg pipe and put on overlay to a video and send it on mpegts using mpeg1video as codec to my localhost server that send data to websocket.

Someone did ever someting similar using jmuxer? as i understand i need a way (codec) that translate in jmuxer packet format

2 bytes | 2 bytes |   |  
-- | -- | -- | --
Duration (ms) | Audio Data Length | Audio Data (AAC) | Video Data (H264)

thanks
Andrea

**Comment by samirkumardas** [comment]
@andreac  The packet format is not mandatory. It was a just example of the packet format.
jMuxer just needs duration and payload. That's all. Even I think you can get it from mpegts packet after parsing it.
